app/views/landing/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="header-color-bar"> </div>
```

Comment:
This div is not quite right. You can style it like this, but it's not an element of the landing page. Yo should style the `logo-section` with something like a `border-top: 5px solid $orange;` so it's just for decoration :smile: